getting-started/traced-example/package.json

```diff
@@ -13,9 +13,8 @@
     "@opentelemetry/exporter-zipkin": "^0.18.2",
     "@opentelemetry/node": "^0.18.2",
     "@opentelemetry/instrumentation": "^0.18.2",
-    "@opentelemetry/plugin-express": "^0.14.0",
-    "@opentelemetry/plugin-http": "^0.18.2",
-    "@opentelemetry/plugin-https": "^0.18.2",
+    "@opentelemetry/instrumentation-express": "^0.15.0",
```

Comment:
Not needed in this PR - but I think we should get rid of dependencies to contrib in this repo.